### ISSUE TITLE
No version of PackageCompiler meets requirements

### ISSUE THREAD

**Issue by jstechND** [kdheepak/JuliaFormatter.vim#53]
(Caveat: I'm new to Julia.)

I installed JuliaFormatter.vim, but it wasn't doing anything. After some poking around, I found the log, which says,
```
[2023-04-03T10:16:35.764] Cannot Instantiate the project! Install dependencies by yourself: `julia --project=/home/john/config/vim/bundle/JuliaFormatter.vim/ -e 'using Pkg; Pkg.add("JuliaFormatter"); Pkg.add("JSON")'`
```

So I tried that, but I get this error:
```
john:JuliaFormatter.vim/ (main) $ julia --project=/home/john/config/vim/bundle/JuliaFormatter.vim/ -e 'using Pkg; Pkg.add("JuliaFormatter"); Pkg.add("JSON")'
   Updating registry at `~/.julia/registries/General`
ERROR: LoadError: Unsatisfiable requirements detected for package PackageCompiler [9b87118b]:
 PackageCompiler [9b87118b] log:
 ├─possible versions are: [0.5.0-0.5.1, 0.6.0-0.6.5, 1.0.0-1.0.2, 1.1.0-1.1.1, 1.2.0-1.2.8, 1.3.0, 1.4.0-1.4.1, 1.5.0, 1.6.0, 1.7.0-1.7.7, 2.0.0-2.0.11, 2.1.0-2.1.5] or uninstalled
 ├─restricted to versions 2 by an explicit requirement, leaving only versions [2.0.0-2.0.11, 2.1.0-2.1.5]
 └─restricted by julia compatibility requirements to versions: [0.5.0-0.5.1, 0.6.0-0.6.5, 1.0.0-1.0.2, 1.1.0-1.1.1, 1.2.0-1.2.8, 1.3.0, 1.4.0-1.4.1] or uninstalled — no versions left
...
```

I'm on Ubuntu 20.04, with Julia 1.4.1 installed, and neovim 0.8.3.

**Comment by kdheepak** [comment]
Thanks for reporting!

Are you able to use with a more recent version of Julia? 1.4 is a few years old now. If not, I’ll try to recreate the issue when I get the chance.

**Comment by jstechND** [comment]
Thanks--I upgraded Julia and that fixed it. I had 1.4 because that's what's available on Ubuntu 20.04. It'd be nice if there were a PPA with more recent versions, but I didn't find one.
